x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/optimizer/LogicalPlanOptimizerTests.java

```diff
@@ -5515,7 +5515,7 @@ record PushdownShadowingGeneratingPlanTestCase(
             ),
             new PushDownEnrich()
         ),
-        // | COMPLETION y=CONCAT(some text, x) WITH inferenceID
+        // | COMPLETION y =CONCAT(some text, x) INTO WITH { "inference_id" : "inferenceID" }
```

Comment:
another nitpicky comment - but there is no INTO right? same as a few lines below

Reply:
Just a bad replacement.